comp/forwarder/defaultforwarder/component.go

```diff
@@ -29,6 +29,15 @@ func Module(params Params) fxutil.Module {
 	)
 }
 
+// ModulWithOptionTMP defines the fx options for this component with an option.
+// This is a temporary function to until configsync is cleanup.
+func ModulWithOptionTMP(option fx.Option) fxutil.Module {
```

Comment:
@truthbk: Can you try to see how to better handle configsync in order to avoid this kind of hack?